### DISCUSSION TITLE
Display chart from right to left

### DISCUSSION THREAD

**Discussion by matpascaud** [patrykandpatrick/vico#583] [Questions]
Hi!

That might not be something popular, but I would like my chart to start from the right and the screen, and scroll from right to left (cf image below). Two chart are displayed: one on the left, one on the right. So I would like the right one starting from the right of the screen.
![chartRight](https://github.com/patrykandpatrick/vico/assets/534304/c121d354-5415-4b5f-a337-cd8cfdd523be)

Is this something that can be done?

**Comment by patrickmichalik** [comment]
Hello! It’s possible to invert the initial scroll position. Are you using Vico 1 or Vico 2, and are you using it with Jetpack Compose or the view system?

**Reply by matpascaud** [reply]
Hello!
I'am using Vico2 with views and discovered ScrollHandler when upgrading alpha version. I will check if I can obtain the expected result.
I will post the result here and the code.
thanks again for the lib and your work.

**Reply by patrickmichalik** [reply]
Sounds good. `initialScroll = Scroll.Absolute.End` should do the trick. Thanks for the kind words!
